QRCode/Source/QRCode.swift

```diff
@@ -179,6 +179,11 @@ open class QRCode: NSObject, AVCaptureMetadataOutputObjectsDelegate {
             return
         }
         
+        guard let videoInput = videoInput else {
```

Comment:
`canAddInput` method below no longer receives an optional `AVCaptureInput` parameter, that's why the optional binding is necessary.

Previous signature was: 
```Swift
func canAddInput(_ input: AVCaptureInput!) -> Bool
```

Current signature:
```Swift
func canAddInput(_ input: AVCaptureInput) -> Bool
```